src/lib.rs

```diff
@@ -21,6 +22,7 @@ mod socket_id;
 mod srt_congest_ctrl;
 mod srt_version;
 mod stats;
+pub mod tokio;
```

Comment:
With this new split, I would appreciate making tokio an optional dependency at some point. 

Probably something for a future PR though.